examples/vqe_noisyopt.py

```diff
@@ -0,0 +1,194 @@
+"""
+VQE with finite measurement shot noise
+"""
+
+from functools import partial
+import numpy as np
+from scipy import optimize
+import optax
+import tensorcircuit as tc
+from tensorcircuit import experimental as E
+from noisyopt import minimizeCompass, minimizeSPSA
```

Comment:
minimizeCompass is never used? It is nice to also include this optimizer for comparison